packages/teams-js/src/internal/communication.ts

```diff
@@ -132,6 +140,20 @@ export function sendAndUnwrap<T>(actionName: string, ...args: any[]): Promise<T>
   return sendMessageToParentAsync(actionName, args).then(([result]: [T]) => result);
```

Comment:
Curious: should developers ever call this function after your change? Do we want all calls up to the host to be passing some form of version number? Maybe this is an intermediate step and eventually functions like this will be removed or altered to pass version numbers?

Reply:
Once this PR gets checked in, I would recommend developers who is developing new capability to call those functions to pass apiVersion number to Host SDKs. If it is possible, I would like to make it be "should" rather than "suggest to" (but it might request me to take a look on each PR in Teams-Js library to make sure developers use those functions who can send out version numbers)

For your last question, yes, this is an intermediate step. Even if developers who checked in capability without using functions in communication to send out api version numbers to Host SDKs, I will edit functions in that capability later, i.e. @MengyiGong 's VisualMedia Capability. At the end of this project, those functions written as 'send..........WithVersion...' will all get removed (or let's say renamed as original function name and remove original function to make sure all of APIs send out request with apiVersion number using the same name API)

Reply:
Talked with Kangxuan, my understanding was correct